Floofbot/Services/EventHandlerService.cs

```diff
@@ -114,6 +115,28 @@ private async Task CheckUserAutoban(IGuildUser user)
                 }
             }
         }
+
+        // rfurry rules gate
+        public async Task RulesGate(SocketMessage msg)
+        {
+            var userMsg = msg as SocketUserMessage;
+            if (msg == null || msg.Author.IsBot)
+                return;
+
+            // rules gate info
+            ulong rulesChannelId = 225980129799700481;
+            ulong rulesServerId = 225980129799700481;
+            ulong readRulesRoleId = 494149550622375936;
+            string rulesBypassString = "yes";
+
+            if (msg.Channel.Id == rulesChannelId && userMsg.Content.ToLower().Contains(rulesBypassString)) 
```

Comment:
What's the rationale behind using `Contains`? Why not use an exact match?

Reply:
The largest issue we are having with the rules gate is due to typos or people writing it in different ways "eg ihavereadtherules"

If we change it to yes, its much easier to do, however, it would also be good if it caught cases such as "yes I agree" or "yes I do" etc

Reply:
Ah, alright. That's a fair way of putting it.